src/simsopt/util/magnetic_axis_helpers.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from simsopt.geo import CurveRZFourier
+
+def compute_on_axis_iota(axis, biotsavart):
+    """
+    Computes the rotational transform on the magnetic axis of a device using a method based on
+    equation (13) of Greene, Journal of Mathematical Physics 20, 1183 (1979); doi: 10.1063/1.524170.
+    This method was shared by Stuart Hudson and Matt Landreman.
+
+    Args:
+        axis: the magnetic axis of a device as a CurveRZFourier.
+        biotsavart: the magnetic field in which the axis lies.
```

Comment:
This function works for any `MagneticField`, not just `BiotSavart`, correct? If so, how about change the variable name from `biotsavart` to something like `field`.